src/compiler/builders.ts

```diff
@@ -15,27 +17,23 @@ const versionedBuilderMap = {
 
 /**
  * Wraps a buildDom function with functionality that every component needs.
- *
- * @param {import('./types').BuildDom} buildDom
- * @return {import('./types').BuildDom}
  */
-function wrap(buildDom) {
+function wrap(buildDom: BuildDom): BuildDom {
   return function wrapper(element) {
-    applyStaticLayout(/** @type {AmpElement} */ (element));
+    applyStaticLayout(element as AmpElement);
     buildDom(element);
     element.setAttribute('i-amphtml-ssr', '');
   };
 }
 
 /**
  * Returns the set of component builders needed to server-render an AMP Document.
- * @param {import('./types').Versions} versions
- * @param {{[version: string]: import('./types').BuilderMap}} builderMap
- * @return {import('./types').BuilderMap}
  */
-export function getBuilders(versions, builderMap = versionedBuilderMap) {
-  /** @type {import('./types').BuilderMap} */
-  const builders = {};
+export function getBuilders(
+  versions: Versions,
+  builderMap = versionedBuilderMap
```

Comment:
The type for `builderMap` is dropped with this change

Reply:
It was inferred (remember, no `any` is implicitly allowed). Just made it explicit tho!